postgres.go

```diff
@@ -158,8 +171,16 @@ func (db DBConf) dump(sb s3Backend, keyPath string) error {
 
 	log.Debugf("Dump file %v ready for writing", dumpFile)
 
-	key := getKey(keyPath)
-	e, err := newEncryptor(key, wr)
+	privateKey, publicKeyList, err := getKeys(publicKeyPath)
+	if err != nil {
+		log.Error("Could not retrieve public key or generate private key")
+
+		return err
+	}
```

Comment:
```suggestion
	privateKey, publicKeyList, err := getKeys(publicKeyPath)
	if err != nil {
		return err
	}
```

No point of logging an error here since `err` will be propagated to main and logged as `fatal`

Reply:
same